EDI/Operation/07_PrepareWorkflow.md

```diff
@@ -0,0 +1,78 @@
+[!!Manage workflows](../../ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+# Prepare workflow
+
+Extend your workflows by adding an action to export the EDIFACT messages. To do this, use a workflow that processes the business document you want to transfer to your EDIFACT message receiver, for example an invoice or a delivery note. 
+
+To integrate the export to the workflow, you need to insert an
+*Execute PHP code* core action. For detailed information on this core action, see [Execute PHP code](../../ActindoWorkFlow/UserInterface/08_CoreActions.md#execute-php-code) in the *Process Orchestration* docmentation. You can insert this action at any position in the workflow in which you have access to the required business document data. Bear in mind that if any errors that occur after the EDIFACT export and before posting for subsequent processes can lead to data inconsistencies that need to be cleaned up.
+
+The following example shows how to export a delivery note via an export definition.
+
+
+#### Prerequisites
+
+- You are familiar with the configuration of workflows. For detailed information, see [Manage the workflows](../../ActindoWorkFlow/Operation/01_ManageWorkflows.md) in the *Process Orchestration* documentation. 
+- The export definition is activated, see [Activate definition](01_ManageDefinitions.md#activate-definition).
```

Comment:
Link geht nicht: (./01_ManageDefinitions.md#activate-definition)

Reply:
Soll auch nicht, sondern in Manage definition